builds/jazz-build-module/environment-deployment-metadata-loader.groovy

```diff
@@ -67,6 +67,29 @@ def getEnvironmentLogicalId() {
 	return g_environment_logical_id
 }
 
+def getEnvironmentInfo() {
+	if (g_service_config['domain'] != "jazz") {
+		def API_ENVIRONMENT_QUERY_URL = "${g_environment_api}?service=${g_service_config['service']}&domain=${g_service_config['domain']}"
+		def getEnvironments = sh(script: "curl -H \"Content-type: application/json\" -H \"Authorization:\"$g_login_token -X GET \"${g_environment_api}?service=${g_service_config['service']}&domain=${g_service_config['domain']}\" ", returnStdout: true).trim()
```

Comment:
We can use API_ENVIRONMENT_QUERY_URL

Reply:
Fixed it. It was copy paste code from another place which had similar issue, so fixed it there too.